submissions/Shramkoweb/html-css-popup/source/sass/blocks/main-nav.scss

```diff
@@ -0,0 +1,83 @@
+.main-nav {
+  position: relative;
+
```

Comment:
Remove empty lines inside code block.

Reply:
I use declaration order and grouped together selectors following the order:

1. Positioning
2. Box model
3. Typographic
4. Visual

For this i include empty lines inside code block.

Reply:
I understand your position but it's not obvious.